CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ option(FLB_PROXY_GO           "Enable Go plugins support"   Yes)
 
 # Build-in Plugins
 option(FLB_IN_CPU             "Enable CPU input plugin"            Yes)
+option(FLB_IN_DBUS            "Enable DBus input plugin"           Yes)
```

Comment:
For upstreaming, since Windows is not currently supported, let's disable the plugin for Windows. Looks like the place to do that is [here](https://github.com/Formlabs/fluent-bit/blob/master/cmake/windows-setup.cmake).